packages/playwright-ct-vue/registerSource.mjs

```diff
@@ -187,6 +206,14 @@ window.playwrightRerender = async (rootElement, options) => {
   if (!component)
     throw new Error('Component was not mounted');
 
-  for (const [key, value] of Object.entries(options.props || {}))
+  const { slots, listeners, props } = createComponent(options);
+
+  component.slots = wrapFunctions(slots);
+  allListeners[0][1] = listeners;
```

Comment:
@pavelfeldman Not sure if this could hurt?

Reply:
I'd rather move from array to Map for allListeners and implemented it strictly.